mapie/tests/test_utils.py

```diff
@@ -38,6 +44,32 @@
 ]
 
 
+prng = RandomState(1234567890)
+y_score = prng.random(51)
+y_scores = prng.random((51, 5))
+y_true = prng.randint(0, 2, 51)
+
+results_binning = {
+    "quantile":
+        [
+            0.03075388, 0.17261836, 0.33281326, 0.43939618,
+            0.54867626, 0.64881987, 0.73440899, 0.77793816,
+            0.89000413, 0.99610621
+        ],
+    "uniform":
+        [
+            0, 0.11111111, 0.22222222, 0.33333333, 0.44444444,
+            0.55555556, 0.66666667, 0.77777778, 0.88888889, 1
+        ],
+    "array split":
+        [
+            0.62689056, 0.74743526, 0.87642114, 0.88321124,
+            0.8916548,  0.94083846, 0.94999075, 0.98759822,
+            0.99610621, np.inf
+        ],
+}
```

Comment:
Same remark as in test_calibration, did you find those numbers by yourself or using your code ?

Reply:
Now I have, same results.